packages/toggle-switch-react/src/ToggleSwitch.tsx

```diff
@@ -25,13 +25,14 @@ export const ToggleSwitch = ({ children, pressed, onClick, className, disabled,
                 disabled={disabled}
                 className={componentClassName}
                 onClick={onClick}
+                data-testid="jkl-toggle-switch"
```

Comment:
Dette er vel muligens rester av mine feil, men greit å få ryddet opp i det.

```suggestion
                role="switch"
                aria-checked={!!pressed}
                data-testid="jkl-toggle-switch"
```

Fjerne aria-pressed.

Også må vi knytte knappen til labelen.

Reply:
Denne er nå implementert som en `<button>` per w3c-anbefaling, så `aria-pressed` er riktig:

> Toggle button: A two-state button that can be either off (not pressed) or on (pressed). To tell assistive technologies that a button is a toggle button, specify a value for the attribute `aria-pressed`.

Siden det nå er en knapp er det ikke behov for label lenger, all tekst er inne i knappen 😊

Reply:
Hmm.. Leste om `role="switch"`nå på MDN. Ser ut som de oppfører seg ganske identisk. Rart at det ikke står nevnt hos w3c. Skal undersøke om det er en av dem som foretrekkes med tanke på UU!